portfolio/src/main/webapp/comment-form.js

```diff
@@ -0,0 +1,20 @@
+// Retrieve and post user comments
+async function getComments() {
+  const response = await fetch('/handle-comment');
+  const comments = await response.json();
+
+  // Properly format and place the fetched comments within the feed
+  const feed = document.getElementById('comment-display'); 
+  feed.innerHTML = '';
+  formatComments(feed, comments);
+}
+
+// Format list of user comments such that each is represented as a paragraph
```

Comment:
These are documentation comments, not implementation, so JSDoc should be used

https://google.github.io/styleguide/jsguide.html#jsdoc